repository/src/main/java/org/apache/atlas/repository/audit/ESBasedAuditRepository.java

```diff
@@ -302,32 +295,14 @@ void createSession() throws AtlasException {
                 LOG.info("Updating ES total field limit");
                 updateFieldLimit();
             }
+            updateMappingsIfChanged();
```

Comment:
what happens when we create the audit index successfully but fail to update the mappings? just curious why we don't update the mapping directly when we create the index?

Reply:
As per my understanding, new instances will have the correct mappings by default, this is to ensure mappings on existing instances get updated